src/components/Repository.js

```diff
@@ -83,6 +83,17 @@ function Repository({match}) {
                 </a>
               </em>
             </small>
+            <div className="languages">
+              {repository && repository.languages.nodes.map((language, key) => (
+                <span key={key}>
+                  <span className="dot"  style={{color: language.color}}>•</span>
+                  <span className="name">{language.name}</span>
+                </span>
+              ))}
+              <span className="more">
+                {repository && repository.languages.totalCount > 3 && `+${repository.languages.totalCount - 3} languages`}
```

Comment:
the number "3" could be stored in a constant/global variable

Reply:
@bdougie WDYT?

Reply:
I agree with @sujitagarwal, should be dynamic

Reply:
I think that is a great idea. If anyone would like to create a PR to this branch addressing this, please feel free. Since this is a release branch, it will be rebased instead of squashed, which is why I prefer the changes on a PR.

Reply:
@bdougie Done